pkg/datapath/connector/veth.go

```diff
@@ -123,6 +123,28 @@ func SetupVethWithNames(lxcIfName, tmpIfName string, mtu int, ep *models.Endpoin
 		return nil, nil, fmt.Errorf("unable to bring up veth pair: %s", err)
 	}
 
+	if groMaxSize > 0 {
+		if err = netlink.LinkSetGROMaxSize(hostVeth, groMaxSize); err != nil {
+			return nil, nil, fmt.Errorf("unable to set GRO max size to %q: %s",
+				lxcIfName, err)
+		}
+		if err = netlink.LinkSetGROMaxSize(peer, groMaxSize); err != nil {
+			return nil, nil, fmt.Errorf("unable to set GRO max size to %q: %s",
```

Comment:
Should we distinguish between the host & peer link in either of these error cases messages?

Reply:
No, we don't care which one failed. This is handled the same for both by the driver, if one fails then the other will fail as well.